### PR TITLE
CSPL-2062: Merge develop to feature readiness

### PR DESCRIPTION
**Pull request by pdhanoyasplunk**
Merge Develop back to feature-readiness-liveness-configurable branch

### CODE REVIEW

**Review by vivekr-splunk — APPROVED** [review]

**Review by sgontla — APPROVED** [review]

File: bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -4,9 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: splunk-operator
-  operators.operatorframework.io.bundle.channels.v1: "beta,stable"
-  operators.operatorframework.io.bundle.channel.default.v1: "stable"
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.21.0
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.15.0
```
Comment (gaurav-splunk) [inline bundle/metadata/annotations.yaml:8]: is this 1.15 or 1.18? @vivekr-splunk ?

File: controllers/clustermanager_controller_test.go [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+		time.Sleep(2 * time.Second)
+	})
+
+	AfterEach(func() {
+
+	})
+
+	Context("ClusterManager Management failed", func() {
+
+		It("Get ClusterManager custom resource should failed", func() {
```
Comment (gaurav-splunk) [inline controllers/clustermanager_controller_test.go:35]: nit: should fail

Comment (pdhanoyasplunk) [reply controllers/clustermanager_controller_test.go:35]: This is coming from develop. https://github.com/splunk/splunk-operator/blob/develop/controllers/clustermanager_controller_test.go#L35

File: bundle/metadata/annotations.yaml (hunk 1 above)
Comment (pdhanoyasplunk) [reply bundle/metadata/annotations.yaml:8]: These values are coming form develop
https://github.com/splunk/splunk-operator/blob/develop/bundle/metadata/annotations.yaml#L7-L8
If we need updates I will push changes on develop first.

**Review by gaurav-splunk — APPROVED** [review]

**Pull request merged**